R/complete.R

```diff
@@ -0,0 +1,48 @@
+#' Complete development session
+#'
+#' helps complete a session by checking if NEWS(.md), version, and date are up-to-date
+#' @param pkg package description, can be path or package name.  See
+#'   \code{\link{as.package}} for more information
+#' @export
+complete <- function(pkg = ".") {
+
+  pkg <- as.package(pkg)
+
+  if (file.exists("NEWS.md")) {
+    rule("Git: last commit message")
+    try( print( readLines(file.path(pkg$path, ".git/COMMIT_EDITMSG"), n = 1) ) )
+    rule("NEWS.md file")
+    print( readLines("NEWS.md", n = 10) )
+    if (yesno("Is NEWS.md file up-to-date?"))
+      return(invisible())
+  } else if (file.exists("NEWS")) {
+    rule("Git: last commit message")
+    try( print( readLines(file.path(pkg$path, ".git/COMMIT_EDITMSG"), n = 1) ) )
+    rule("NEWS file")
+    try(print(show_news(pkg)))
+    rule("NEWS file up-to-date")
+    if (yesno("Is NEWS file up-to-date?"))
+      return(invisible())
+  }
+
+  rule("Version number")
+  print(pkg$version)
+  if (yesno("Is the version number up to date?"))
+    return(invisible())
+
+  if ( exists("date", envir = 1) & pkg$date != Sys.Date() ) {
```

Comment:
I'm not a big fan of including the `Date` field in the DESCRIPTION, just because it's something you need to keep up-to-date, and doesn't really seem necessary (since a built date is automatically added when you build the package). Do you have a reason you like to use it?